### PR TITLE
Add Providers to the list view of ClusterOutputs

### PR DESCRIPTION
**Pull request by codyrancher**
rancher/dashboard#1568

Screen Shot 2020-10-05 at 1 30 11 PM
Screen Shot 2020-10-05 at 1 30 03 PM

### CODE REVIEW

File: config/product/logging.js [hunk 1]
```diff
@@ -36,4 +38,7 @@ export function init(store) {
     route:      { name: 'c-cluster-logging' },
     exact:       true,
   });
+
+  headers(LOGGING.OUTPUT, [STATE, NAME_COL, LOGGING_OUTPUT_PROVIDERS, AGE]);
```
Comment (vincent99) [inline config/product/logging.js:42]: These are missing the namespace column (after name)

**Review by vincent99 — CHANGES_REQUESTED** [review]

File: config/product/logging.js (hunk 1 above)
Comment (codyrancher) [reply config/product/logging.js:42]: Do we actually want that? It's grouped by namespace.

Comment (vincent99) [reply config/product/logging.js:42]: Grouping by namespace should be optional (like it is on every other type).  Which works by removing the `NAMESPACE` column from the list if grouping is enabled.

Comment (codyrancher) [reply config/product/logging.js:42]: I understand that it's optional. I'm trying to understand where this request is coming from.

I think my confusion is stemming from the fact that the NAMESPACE column isn't actually working on any of the types that it's been added to.

I navigated to Ingress, secret, workload etc and each was grouped by namespace and didn't have a namespace column. I just looked at explorer.js and noticed that a lot of these types are actually supposed to have this column.

I'll look into fixing that.

Comment (codyrancher) [reply config/product/logging.js:42]: Nevermind, I see that only specific cases will show the column.

**Comment by codyrancher** [comment]
Added the namespace column to both types. Visually there's no difference however.

**Pull request merged**